tests/conftest.py

```diff
@@ -0,0 +1,130 @@
+from jinja2 import Environment
```

Comment:
Let's try to minimize local namespace pollution and stick with "import x" form for most part, so at usage point it will be clear where this symbol is coming from. From this list only "OpenSSL" deserves shortcut, as "crypto" is quite self-descriptive.

Reply:
OK, I don't mind. Fixed.